metabase/metabase
# Change api endpoint used to load collection content for SavedQuestionList

Closes https://github.com/metabase/metabase/issues/24660

### Description

When sub-collections has the same name in several parent-collections, then the Saved Question picker will show all questions from all the sub-collections, which can be confusing.

### How to verify

To Reproduce

1. Create two parent collections, "Parent1" and "Parent2"
2. Then create a sub-collection within both parents called "same-child-name"
3. Create two questions, save "Q1" to "Parent1/same-child-name" and save "Q2" to "Parent2/same-child-name"
4. New question > Saved Questions > "Parent1" > "same-child-name" - it should show only question Q1.

### Demo

https://user-images.githubusercontent.com/7983744/228951960-e3ee5213-cafe-4e47-ae9e-70c72bc04cdb.mov

### Checklist



- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/metabase/metabase/29698)
